src/pages/our_ambassadors.astro

```diff
@@ -16,7 +16,15 @@ import AmbassadorCard from "@components/AmbassadorCard/index.tsx";
         linkedin="abhi18av"
         twitter="abhi18av"
       />
-
+      <AmbassadorCard
```

Comment:
Please, place your ambassador card in alphabetical order, together with the other ambassadors whose first names start with **i**.